core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -347,6 +347,20 @@ default boolean isComplete() {
      */
     CompletableFuture<Void> whenComplete();
 
+    /**
+     * Requests to start streaming data to {@link NoopSubscriber}.
+     *
+     * <p>For example:<pre>{@code
+     * StreamMessage<Integer> source = StreamMessage.of();
+     * CompletableFuture<Void> future = source.peek(x-> {}).subscribe();
+     * assert future.isDone().equals(true);
+     * }</pre>
+     */
+    default CompletableFuture<Void> subscribe() {
```

Comment:
Q) What do you think of failing the CF if the stream fails?

e.g.
```
final CompletableFuture<Void> cf = StreamMessage.aborted(new RuntimeException()).subscribe();
// wait for cf completion
assertThat(cf).isCompletedExceptionally();
```

Reply:
It looks simpler! I'll apply it😄

Reply:
Ah sorry 😅  This was a bad review, I think the previous javadoc is fine.
Do you mind reverting the javadoc changes for ca884e8fc27c9f7d2aebc3238ed2fda8f5fca799

```
     * StreamMessage<Integer> source = StreamMessage.of();
     * CompletableFuture<Void> future = source.peek(x-> {}).subscribe();
     * assert future.isDone().equals(true);
```